components/homeComponents/HomeHero.tsx

```diff
@@ -0,0 +1,37 @@
+import { useRouter } from 'next/router';
+import { buttonDatas } from '../../lib/data';
+
+export default function HomeHero() {
+  const router = useRouter();
+
+  return (
+    <section className="min-h-screen p-4 bg-contain bg-white">
+      <div
+        style={{ minHeight: 480 }}
+        className="max-w-4xl mx-auto flex flex-col justify-center items-center"
+      >
+        <h1 className="text-center md:text-8xl text-6xl font-bold bg-clip-text text-transparent text-primaryDark">
```

Comment:
can u remove the `text-transparent` & `bg-clip-text` here? I added it for the gradient version kelly made and forgot remove it when she changed.

Reply:
@thetechie7 can you make a PR to do it? That is not relevant to this PR.